packages/netlify-cms-widget-number/src/NumberControl.js

```diff
@@ -100,19 +107,29 @@ export default class NumberControl extends React.Component {
     const min = field.get('min', '');
     const max = field.get('max', '');
     const step = field.get('step', field.get('value_type') === 'int' ? 1 : '');
+
+    const prefix = field.get('prefix', false);
+    const suffix = field.get('suffix', false);
+
     return (
-      <input
-        type="number"
-        id={forID}
-        className={classNameWrapper}
-        onFocus={setActiveStyle}
-        onBlur={setInactiveStyle}
-        value={value || (value === 0 ? value : '')}
-        step={step}
-        min={min}
-        max={max}
-        onChange={this.handleChange}
-      />
+      <div className={classNameWrapper}>
+        <div css={innerWrapper}>
+          {prefix && (<span>{prefix}&nbsp;</span>)}
+          <input
+            type="number"
+            id={forID}
+            onFocus={setActiveStyle}
+            onBlur={setInactiveStyle}
+            value={value || (value === 0 ? value : '')}
+            step={step}
+            min={min}
+            max={max}
+            onChange={this.handleChange}
+            css={css`flex-grow: 1`}
+          />
+          {suffix && (<span>&nbsp;{suffix}</span>)}
```

Comment:
wondering if onFocus and onBlur should go on the bigger div (classed {classNameWrapper})